openapi.yaml

```diff
@@ -4722,7 +4711,9 @@ components:
             (link to older version) and `successor-version` (link to newer version) can also be used
             to show the relation between versions.
 
-            2. `cite-as`: For all DOIs associated with the process, the respective DOI
+            2. `example`: Used to refer to examples of user-defined processes that use this process.
```

Comment:
What does the example link as process graph show? Is it an alternative implementation of the process or is it an example how it is used in practice? What about authorization restrictions, when linking to an user defined graph?

Reply:
> What does the example link as process graph show?

A JSON file with a process in JSON format (i.e. a JSON object with at least the member `process_graph` and its nodes).

> Is it an alternative implementation of the process or is it an example how it is used in practice?

An example.

> What about authorization restrictions, when linking to an user defined graph?

Maybe I should remove the `user-defined` here. It's a process provided by the back-end as an example.

Reply:
```suggestion
            2. `example`: Links to examples of other processes that use this process.
```